packages/ai-constructs/src/conversation/runtime/bedrock_converse_adapter.test.ts

```diff
@@ -70,315 +75,702 @@ void describe('Bedrock converse adapter', () => {
     }
   );
 
-  void it('calls bedrock to get conversation response', async () => {
-    const event: ConversationTurnEvent = {
-      ...commonEvent,
-    };
+  [false, true].forEach((streamResponse) => {
+    // This is a common set of use cases that both streaming and non-streaming version must support.
+    void describe(`${streamResponse ? 'with' : 'without'} streaming`, () => {
+      void it('calls bedrock to get conversation response', async () => {
```

Comment:
The majority of changes here are to adapt existing tests to handle both streaming and non streaming calls.

The reason for this is that:
1. we want to have functional parity between streaming and non-streaming
2. at least 2/3 is common - e.g. mocking bedrock inputs.
3. the form of output is different, but it carries more-less same content.

These changes contribute most to the volume of this PR.